pyproject.toml

```diff
@@ -13,7 +13,7 @@ maintainers = [
     {name = "Felix Dittrich"},
 ]
 readme = "README.md"
-requires-python = ">=3.8.0,<4"
+requires-python = ">=3.9.0,<4"
```

Comment:
@odulcy-mindee as alternative we could keep 3.8 here for the moment until you have internal also the 3.9 upgrade done wdyt ?
But all the tests should switch already to 3.9 as new "main" version